Add input mapping to creeps tutorial (#5383)
godotengine/godot-docs: getting_started/first_2d_game/03.coding_the_player.rst

we have 4 direction inputs to check. Input actions are defined in the Project
Settings under "Input Map". Here, you can define custom events and assign
different keys, mouse events, or other inputs to them. For this game, we will

> map the WASD keys to the four directions.

Reviewer: I think it's better to stick to arrow keys for this tutorial. That makes the tutorial valid for any keyboard layout, instead of only QWERTY.

Reviewer: Alternatively it could use WASD with Physical Keys, but I think that's a nuance better covered in a dedicated input mapping tutorial, so here I'd keep it simple and use arrow keys.

Author: Okay, that makes sense.  I can rework it and use the arrow keys rather than WASD.

we have 4 direction inputs to check. Input actions are defined in the Project
Settings under "Input Map". Here, you can define custom events and assign
different keys, mouse events, or other inputs to them. For this game, we will
map the arrow keys to the four directions.

> Click on *Project -> Project Settings* to open the project settings and then
> click on the *Input Map* tab at the top. Enter "move_right" and then click
> the "Add" button to add the ``move_right`` action, as shown here:

Reviewer 2: ```suggestion
Click on *Project -> Project Settings* to open the project settings window and
click on the *Input Map* tab at the top. Type "move_right" in the top bar and
click the "Add" button to add the ``move_right`` action.
```

.. image:: img/input-mapping-add-action.png

> Now we need to add a key assignment to this action. Press the "+" button and
> then press "Key" in the pop-up menu.  A dialog will ask you to press the key
> you want mapped.  Press the right arrow and then click "Ok":

Reviewer 2: ```suggestion
We need to assign a key to this action. Click the "+" icon on the right, then
click the "Key" option in the drop-down menu. A dialog asks you to type in the
desired key. Press the right arrow on your keyboard and click "Ok".
```

Author: Ugh, I tried to be careful about the words I used.  I'll get this right eventually.  😆

.. image:: img/input-mapping-add-key.png

> Repeat these steps to add three more mappings: ``move_left`` mapped to the left arrow
> key, ``move_up`` mapped to the up arrow key, and ``move_down`` mapped to the down arrow key.
> It should look something like this:

Reviewer 2: ```suggestion
Repeat these steps to add three more mappings:

1. ``move_left`` mapped to the left arrow key.
2. ``move_up`` mapped to the up arrow key.
3. And ``move_down`` mapped to the down arrow key.

Your input map tab should look like this:
```

.. image:: img/input-mapping-completed.png

> Press "Close" to close the project settings.

Reviewer 2: ```suggestion
Click the "Close" button to close the project settings.
```

> .. note::
> 
>   You can map multiple keys, joystick buttons, or joystick
>   axes to the same input action.

Reviewer 2: ```suggestion
.. note::

   We only mapped one key to each input action, but you can map multiple keys,
   joystick buttons, or mouse buttons to the same input action.
```
